pages/veteran-representative/index.md

```diff
@@ -1,7 +0,0 @@
----
-title: Appoint VSO as representative
-entryname: veteran-representative
-layout: page-react.html
-description: Appoint a Veterans Service Officer to represent you.
-vagovprod: false
----
```

Comment:
Migrated here, https://github.com/department-of-veterans-affairs/vets-website/pull/10660/files#diff-85cff28c0dfb618c31511c3e06275f5aR5. __This app is disabled due to feature flag.__